### ISSUE TITLE
[Bug]: Images not parsing properly

### ISSUE THREAD

**Issue by tmcelrath** [SpeciesFileGroup/taxonworks#3443]
### Steps to reproduce the bug

```bash
1. Certain uploaded images are not parsing properly
2. E.g. https://sfg.taxonworks.org/tasks/accessions/breakdown/sqed_depiction/176130 
3. Original image: https://sfg.taxonworks.org/system/images/image_files/000/871/845/original/img_0026.jpg?1684865223 (looks rotated properly)
4. SQED depiction: https://sfg.taxonworks.org/depictions/927294
5. I can download the image, save it in Mac OS X preview (doing nothing else), and then reupload.
6. Then the image recalculated correctly.
```

### Expected behavior

Image should calculate correctly. 

There are several 1000 images that are affected. 

### Environment

Production

### Version

0.33.1

### Browser Used

Chrome

**Comment by tmcelrath** [comment]
@mjy

**Comment by mjy** [comment]
Fixed, duplicate of #3070
